x-pack/plugins/siem/public/components/page/network/domains_table/index.tsx

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBadge, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { isEqual, last } from 'lodash/fp';
+import React from 'react';
+import { connect } from 'react-redux';
+import { ActionCreator } from 'redux';
+import styled from 'styled-components';
+
+import {
+  Direction,
+  DomainsEdges,
+  DomainsFields,
+  DomainsSortField,
+  FlowDirection,
+  FlowTarget,
+} from '../../../../graphql/types';
+import { networkActions, networkModel, networkSelectors, State } from '../../../../store';
+import { FlowDirectionSelect } from '../../../flow_controls/flow_direction_select';
+import { Criteria, ItemsPerRow, LoadMoreTable, SortingBasicTable } from '../../../load_more_table';
+
+import { getDomainsColumns } from './columns';
+import * as i18n from './translations';
+
+interface OwnProps {
+  data: DomainsEdges[];
+  flowTarget: FlowTarget;
+  loading: boolean;
+  hasNextPage: boolean;
+  ip: string;
+  nextCursor: string;
+  totalCount: number;
+  loadMore: (cursor: string) => void;
+  endDate: number;
+  startDate: number;
+  type: networkModel.NetworkType;
+}
+
+interface DomainsTableReduxProps {
+  domainsSortField: DomainsSortField;
+  flowDirection: FlowDirection;
+  limit: number;
+}
+
+interface DomainsTableDispatchProps {
+  updateDomainsDirection: ActionCreator<{
+    flowDirection: FlowDirection;
+    networkType: networkModel.NetworkType;
+  }>;
+  updateDomainsLimit: ActionCreator<{
+    limit: number;
+    networkType: networkModel.NetworkType;
+  }>;
+  updateDomainsSort: ActionCreator<{
+    domainsSort: DomainsSortField;
+    networkType: networkModel.NetworkType;
+  }>;
+}
+
+type DomainsTableProps = OwnProps & DomainsTableReduxProps & DomainsTableDispatchProps;
+
+const rowItems: ItemsPerRow[] = [
+  {
+    text: i18n.ROWS_5,
+    numberOfRow: 5,
+  },
+  {
+    text: i18n.ROWS_10,
+    numberOfRow: 10,
+  },
+  {
+    text: i18n.ROWS_20,
+    numberOfRow: 20,
+  },
+  {
+    text: i18n.ROWS_50,
+    numberOfRow: 50,
+  },
+];
+
+const CountBadge = styled(EuiBadge)`
+  margin-left: 5px;
+`;
```

Comment:
Optional, if we have a spot to put this in to share its use that would be great. This `CountBage` Ironically enough could count that it, its self is in copied 3 different times.

![count](https://user-images.githubusercontent.com/1151048/56064434-3c6af780-5d2f-11e9-9ec2-e7795295b6c1.jpg)

Reply:
This is great, ha! ^_^ 

I went ahead and moved `CountBadge` to `public/components/page/index.tsx` and updated all references, which looks to be at **4** actually. 1, 2, 3.... **4**. 🍪

I also opened https://github.com/elastic/ingest-dev/issues/380, as that file contains quite a few stale common components.